shell.go

```diff
@@ -831,3 +852,34 @@ func (s *Shell) ObjectStat(key string) (*ObjectStats, error) {
 
 	return stat, nil
 }
+
+// ObjectStat gets stats for the DAG object named by key. It returns
+// the stats of the requested Object or an error.
+func (s *Shell) StatsBW() (*p2pmetrics.Stats, error) {
+	v := &p2pmetrics.Stats{}
+	err := s.RequestDecode(context.TODO(), &v, "stats/bw")
```

Comment:
Other functions use.`.Background`

Reply:
Even better, let's just take a context (it's a new command).